### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by crushcat**
fix of issues

### CODE REVIEW

File: publics/ext.js [hunk 1]
```diff
@@ -1,205 +1,204 @@
...
-        if (this.readyState != 4)
-            return;
-        if (this.status != 200) {
-            return;
-        } else {
-            var items = JSON.parse(this.responseText);
-            if (!!items) {
-                refreshItems(items);
-            }
+'use strict';
```
Comment (mikalai-sauchanka) [inline publics/ext.js:1]: Wrap the file into self executing function to prevent global object pollution

Comment (crushcat) [reply publics/ext.js:1]: fixed in  https://github.com/crushcat/epam-tasks-app2/pull/2/commits/6d0bb49857ba47fac3202b091f52d9eb3d46576e

Comment (crushcat) [reply publics/ext.js:1]: re-fixed in https://github.com/crushcat/epam-tasks-app2/pull/2/commits/87a3aea2702d6ed38d4f3a6b72de4d15d263b9fa

Comment (crushcat) [reply publics/ext.js:1]: formatting in https://github.com/crushcat/epam-tasks-app2/pull/2/commits/cd0c16131952159dcf31c11c1db620597dd1543c

**Comment by mikalai-sauchanka** [comment]
Ok.

**Pull request merged**